src/components/Layouts/ForgottonPasswordContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import backgroundImage from '../../assets/register-login-background.svg';
+import logo from '../../assets/otherLogo.svg';
+import { ReactNode } from 'react';
```

Comment:
we no longer handing asset in this way, check the file called image and import logo from there. and also always organize your import.

Reply:
let me change it